integration/init_test.go

```diff
@@ -104,9 +107,19 @@ func TestIntegration(t *testing.T) {
 
 	SetDefaultEventuallyTimeout(30 * time.Second)
 
+	builder, err = pack.Builder.Inspect.Execute()
+	Expect(err).NotTo(HaveOccurred())
+
 	suite := spec.New("Integration", spec.Report(report.Terminal{}), spec.Parallel())
-	suite("Layer Reuse", testLayerReuse)
-	suite("OfflineApp", testOffline)
-	suite("SimpleApp", testSimpleApp)
+	// This test will only run on the Bionic full stack, because stack upgrade
+	// failures have only been observed when upgrading from the Bionic full stack.
+	// All other tests will run against the Bionic base stack
+	if builder.BuilderName == "paketobuildpacks/builder:buildpackless-full" {
```

Comment:
You'll want to use the Stack name, not the builder name here, otherwise you won't be able to run these tests on any other builder.

Reply:
In this scenario, I only want to run this test on this builder though